gensim/test/test_phrases.py

```diff
@@ -646,6 +646,19 @@ def testEncoding(self):
         self.assertTrue(isinstance(transformed, six.text_type))
 
 
+class TestPhraserModelCompatibilty(unittest.TestCase):
+
+    def TestCompatibilty(self):
+        with temporary_file("phraser_model_3dot6") as fpath:
+            bigram_loaded = Phraser.load(fpath)
+            test_sentences = [u'trees', u'graph', u'minors']
+            prev_ver = bigram_loaded(test_sentences)
+            phrase_model = Phrases(test_sentences, threshold=1)
+            bigram = Phraser(phrase_model)
+            curr_ver = bigram(test_sentences)
```

Comment:
but `Phraser` isn't callable, `[test_sentences]` should be used here (same for 655). How this pass tests?